01-JS-DOM-APIs/EX01/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+   	<title> JS 01 </title>
+   	<meta name="description" content="Let's begin with JavaScript">
+   	<meta name="viewport" content="width=device-width, initial-scale=1">
+   	<link rel="stylesheet" href="css/normalize.css">
+      <link rel="stylesheet" href="css/styles.css">
+      <link href="https://fonts.googleapis.com/css?family=Bubbler+One" rel="stylesheet">
+   </head>
+   <body>
+   	<header>
+   		<h1>This it my first JS excercise!</h1>
+   	</header>
+   	<section>
+   		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam ac sem porttitor, ultricies ipsum non, feugiat leo. Fusce volutpat tristique rutrum. Phasellus blandit vehicula pretium. Donec scelerisque mauris ac gravida bibendum. Vestibulum quis maximus purus, ut vestibulum elit. Nullam velit lacus, commodo sed sapien eget, eleifend pellentesque arcu. Mauris a lorem in orci rhoncus aliquet sed vel justo. Vestibulum tincidunt neque venenatis, lobortis eros sed, accumsan urna. Ut vel magna euismod, convallis nibh eget, sagittis nulla. Ut a tellus libero. Praesent vestibulum elit luctus augue tristique, ac ullamcorper orci venenatis. Donec et sollicitudin massa. Cras leo leo, luctus at elit a, commodo faucibus velit.</p>
+   	</section>
+   	<section id="hide_Show">
+   		<h2>Hello World!</h2>
+   	</section>
+   	<section>
+   		<p>Sed eu volutpat risus. Pellentesque laoreet vel nisi nec tempus. Nullam id mauris ut metus ultricies vehicula vitae et tellus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Aenean a est sem. Proin eget elementum mi. Nulla iaculis accumsan massa, vel elementum purus vestibulum nec. Mauris fringilla ipsum sit amet dui elementum, in mattis ante hendrerit. Curabitur massa nisi, feugiat eget neque vitae, egestas lacinia felis. Nullam porta, ex sit amet malesuada finibus, arcu nunc scelerisque ante, id fermentum nulla mauris facilisis arcu. Nulla sit amet justo rhoncus, pretium risus in, tristique mauris. Pellentesque at enim venenatis, tempor eros sit amet, feugiat felis. Donec mauris augue, molestie sed urna vel, elementum dictum dui. Cras luctus justo ante, a dapibus enim placerat sit amet. Fusce accumsan libero eget arcu scelerisque sagittis. Pellentesque nec nisl eros.</p>
+   	</section>
+
+   	<script>
+			document.getElementById("hide_Show").style.visibility = "visible";
```

Comment:
Probably you won't see any change once the page is loaded. I mean you will always see the section#hide_show visible. Try to change the approach to use opacity and wait till the page is loaded to make the style update.